ddoc.dd

```diff
@@ -684,43 +684,48 @@ $(P
 )
 
 $(DDOCCODE
-B =     $(LT)b$(GT)$(DOLLAR)0$(LT)/b$(GT)
-I =     $(LT)i$(GT)$(DOLLAR)0$(LT)/i$(GT)
-U =     $(LT)u$(GT)$(DOLLAR)0$(LT)/u$(GT)
-P =     $(LT)p$(GT)$(DOLLAR)0$(LT)/p$(GT)
-DL =    $(LT)dl$(GT)$(DOLLAR)0$(LT)/dl$(GT)
-DT =    $(LT)dt$(GT)$(DOLLAR)0$(LT)/dt$(GT)
-DD =    $(LT)dd$(GT)$(DOLLAR)0$(LT)/dd$(GT)
-TABLE = $(LT)table$(GT)$(DOLLAR)0$(LT)/table$(GT)
-TR =    $(LT)tr$(GT)$(DOLLAR)0$(LT)/tr$(GT)
-TH =    $(LT)th$(GT)$(DOLLAR)0$(LT)/th$(GT)
-TD =    $(LT)td$(GT)$(DOLLAR)0$(LT)/td$(GT)
-OL =    $(LT)ol$(GT)$(DOLLAR)0$(LT)/ol$(GT)
-UL =    $(LT)ul$(GT)$(DOLLAR)0$(LT)/ul$(GT)
-LI =    $(LT)li$(GT)$(DOLLAR)0$(LT)/li$(GT)
-BIG =   $(LT)big$(GT)$(DOLLAR)0$(LT)/big$(GT)
-SMALL = $(LT)small$(GT)$(DOLLAR)0$(LT)/small$(GT)
-BR =    $(LT)br$(GT)
-LINK =  $(LT)a href="$(DOLLAR)0"$(GT)$(DOLLAR)0$(LT)/a$(GT)
-LINK2 = $(LT)a href="$(DOLLAR)1"$(GT)$(DOLLAR)+$(LT)/a$(GT)
-LPAREN= $(LPAREN)
-RPAREN= $(RPAREN)
-DOLLAR= $(DOLLAR)
-DEPRECATED=$(DOLLAR)0
-
-RED =   $(LT)font color=red$(GT)$(DOLLAR)0$(LT)/font$(GT)
-BLUE =  $(LT)font color=blue$(GT)$(DOLLAR)0$(LT)/font$(GT)
-GREEN = $(LT)font color=green$(GT)$(DOLLAR)0$(LT)/font$(GT)
-YELLOW =$(LT)font color=yellow$(GT)$(DOLLAR)0$(LT)/font$(GT)
-BLACK = $(LT)font color=black$(GT)$(DOLLAR)0$(LT)/font$(GT)
-WHITE = $(LT)font color=white$(GT)$(DOLLAR)0$(LT)/font$(GT)
-
-D_CODE =    $(LT)pre class="d_code"$(GT)$(DOLLAR)0$(LT)/pre$(GT)
-D_COMMENT = $(DOLLAR)(GREEN $(DOLLAR)0)
-D_STRING  = $(DOLLAR)(RED $(DOLLAR)0)
-D_KEYWORD = $(DOLLAR)(BLUE $(DOLLAR)0)
-D_PSYMBOL = $(DOLLAR)(U $(DOLLAR)0)
-D_PARAM   = $(DOLLAR)(I $(DOLLAR)0)
+B          = $(LT)b$(GT)$(DOLLAR)0$(LT)/b$(GT)
+I          = $(LT)i$(GT)$(DOLLAR)0$(LT)/i$(GT)
+U          = $(LT)u$(GT)$(DOLLAR)0$(LT)/u$(GT)
+P          = $(LT)p$(GT)$(DOLLAR)0$(LT)/p$(GT)
+DL         = $(LT)dl$(GT)$(DOLLAR)0$(LT)/dl$(GT)
+DT         = $(LT)dt$(GT)$(DOLLAR)0$(LT)/dt$(GT)
+DD         = $(LT)dd$(GT)$(DOLLAR)0$(LT)/dd$(GT)
+TABLE      = $(LT)table$(GT)$(DOLLAR)0$(LT)/table$(GT)
+TR         = $(LT)tr$(GT)$(DOLLAR)0$(LT)/tr$(GT)
+TH         = $(LT)th$(GT)$(DOLLAR)0$(LT)/th$(GT)
+TD         = $(LT)td$(GT)$(DOLLAR)0$(LT)/td$(GT)
+OL         = $(LT)ol$(GT)$(DOLLAR)0$(LT)/ol$(GT)
+UL         = $(LT)ul$(GT)$(DOLLAR)0$(LT)/ul$(GT)
+LI         = $(LT)li$(GT)$(DOLLAR)0$(LT)/li$(GT)
+BIG        = $(LT)big$(GT)$(DOLLAR)0$(LT)/big$(GT)
+SMALL      = $(LT)small$(GT)$(DOLLAR)0$(LT)/small$(GT)
+BR         = $(LT)br$(GT)
+LINK       = $(LT)a href="$(DOLLAR)0"$(GT)$(DOLLAR)0$(LT)/a$(GT)
+LINK2      = $(LT)a href="$(DOLLAR)1"$(GT)$(DOLLAR)+$(LT)/a$(GT)
+LPAREN     = $(LPAREN)
+RPAREN     = $(RPAREN)
+BACKTICK   = $(BACKTICK)
+DOLLAR     = $(DOLLAR)
+DEPRECATED = $(DOLLAR)0
+LT         = $(LT)
+GT         = $(GT)
+AMP        = $(AMP)
+
+RED    = $(LT)font color=red$(GT)$(DOLLAR)0$(LT)/font$(GT)
```

Comment:
This should be raised as another ticket as the goal of this one is to add missing symbols.